app/containers/ModalStack.js

```diff
@@ -25,6 +27,12 @@ const ModalContent = ({ type, closeModal }) => {
     case 'REQUEST_FORM':
       return <Request width={9 / 16} mx="auto" />
 
+    case 'RECEIVE_MODAL':
+      return <ReceiveModal width={9 / 16} mx="auto" />
+
+    case 'ACTIVITY_MODAL':
+      return <ActivityModal width={9 / 16} mx="auto" />
```

Comment:
so the positioning props are already being passed in,. the just aren't being applied correctly.

Reply:
updated